### PR TITLE
Alignment viewer improvements

### PR DESCRIPTION
**Pull request by jackluo**
Closes #[issue number; if this PR doesn't address an issue, delete this line] 

## About 
- [ ] This is a new component 
- [x] I am adding a feature to an existing component, or improving an existing feature
- [ ] I am closing an issue 

## Description of changes 
Added customization on the dash app

## Before merging
- [x] I have gone through the [code review checklist](https://github.com/plotly/dash-component-boilerplate/blob/master/%7B%7Bcookiecutter.project_shortname%7D%7D/review_checklist.md)
- [x] I have completed all of the [steps to take before merging](https://github.com/plotly/dash-bio/blob/master/.github/before_merging.md)
- [x] I know how to [deploy to DDS](https://github.com/plotly/dash-bio/blob/master/.github/deploying_to_dds.md)

### CODE REVIEW

**Comment by shammamah** [comment]
@jackluo Please fix the small CircleCI error!

**Comment by jackluo** [comment]
@shammamah passed :)

**Comment by jackluo** [comment]
Sounds good ! On it

**Comment by jackluo** [comment]
Done, lmk how it looks

File: tests/dashbio_demos/app_alignment_viewer.py [hunk 1]
```diff
@@ -149,14 +200,16 @@ def layout():
...
-                                        "Hover/Click/Event Data"
-                                    ),
-                                    dcc.Textarea(
-                                        id="alignment-events-2",
-                                        placeholder="Hover or click on data to see it here.",
-                                        value="Hover or click on data to see it here.",
-                                        className="alignment-events",
+                                        "Download Sample Data",
                                     ),
+                                    html.A(
```
Comment (xhlulu) [inline tests/dashbio_demos/app_alignment_viewer.py]: Really good idea to let the user download the sample data!

File: tests/dashbio_demos/app_alignment_viewer.py [hunk 2]
```diff
@@ -196,11 +249,262 @@ def layout():
...
                             children=[
                                 html.Div([
-                                    html.H4('Work in progress')
+                                    html.H4('Viewer'),
+                                    html.Hr(className='alignment-separator'),
+                                    html.Div(
+                                        className='alignment-settings',
+                                        children=[
+                                            html.H6("Colorscale"),
+                                            html.P("Choose color theme of the viewer."),
```
Comment (xhlulu) [inline tests/dashbio_demos/app_alignment_viewer.py]: You can choose themes here pretty cool!

File: tests/dashbio_demos/app_alignment_viewer.py [hunk 3]
```diff
@@ -239,13 +543,13 @@ def update_storage(dropdown, contents, filename):
...
         return str(data)
 
-    # Handle event data
-    @app.callback(
-        Output("alignment-events-2", "value"),
-        [Input("alignment-chart", "eventDatum")]
-    )
-    def event_data_select_2(data):
-        return str(data)
+    # # Handle event data
```
Comment (xhlulu) [inline tests/dashbio_demos/app_alignment_viewer.py]: Any reason why this is commented out?

**Pull request merged**